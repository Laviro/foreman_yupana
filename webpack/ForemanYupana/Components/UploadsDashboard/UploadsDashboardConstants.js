export const UPLOADSDASHBOARD_CHANGE_BOOL = 'UPLOADSDASHBOARD_CHANGE_BOOL';
