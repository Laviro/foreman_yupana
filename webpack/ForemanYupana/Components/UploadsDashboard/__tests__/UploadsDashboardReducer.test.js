import { testReducerSnapshotWithFixtures } from 'react-redux-test-utils';

import { UPLOADSDASHBOARD_CHANGE_BOOL } from '../UploadsDashboardConstants';
import reducer from '../UploadsDashboardReducer';

const fixtures = {
  'should return the initial state': {},
  'should handle UPLOADSDASHBOARD_CHANGE_BOOL': {
    action: {
      type: UPLOADSDASHBOARD_CHANGE_BOOL,
      payload: {
        bool: true,
      },
    },
  },
};

describe('UploadsDashboard reducer', () =>
  testReducerSnapshotWithFixtures(reducer, fixtures));
