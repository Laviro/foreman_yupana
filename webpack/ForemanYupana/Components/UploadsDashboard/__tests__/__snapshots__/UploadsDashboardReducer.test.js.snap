// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UploadsDashboard reducer should handle UPLOADSDASHBOARD_CHANGE_BOOL 1`] = `
Object {
  "bool": true,
}
`;

exports[`UploadsDashboard reducer should return the initial state 1`] = `
Object {
  "bool": false,
}
`;
